conftest.py

```diff
@@ -79,12 +83,13 @@ def small_cluster(request):
 
 
 @pytest.fixture
-def small_client(small_cluster):
+def small_client(small_cluster, request):
     with Client(small_cluster) as client:
         small_cluster.scale(10)
         client.wait_for_workers(10)
         client.restart()
-        yield client
+        with coiled.performance_report(f"{request.node.originalname}-{UNIQUE_ID}.html"):
```

Comment:
With this, we are storing the performance reports on coiled, but I can't seem to find which is the bucket or storage that contains them. We should also maybe think about how to clean this up. 

I see that in the coiled account we have some performance reports but not all of them, not sure exactly what's happening

Reply:
> I can't seem to find which is the bucket or storage that contains them

I chatted with @shughes-uk offline and she mentioned today all performance reports are stored in a centralized S3 bucket that only Coiled staff have access to. I don't think this is a blocker for us, but it is good to know. 

> We should also maybe think about how to clean this up.

@shughes-uk mentioned this shouldn't become a problem unless we create millions of performance reports. I'm inclined to not worry about this for now -- I think it will take us a while before we've created millions of these : )

Reply:
> I see that in the coiled account we have some performance reports but not all of them, not sure exactly what's happening

@koverholt pointed out that there's currently a limit of 5 hosted performance reports per user. Per the [`coiled.performance_report` API docs](https://docs.coiled.io/user_guide/api.html#coiled.performance_report)

> Note each user is limited to 5 hosted reports with each a maximum file size of 10mb.

So if we want to keep performance reports around, then we should upload them to our own S3 bucket